com.sap.cloud.lm.sl.cf.persistence/src/main/java/com/sap/cloud/lm/sl/cf/persistence/services/ObjectStoreFileStorage.java

```diff
@@ -101,20 +99,16 @@ public int deleteFilesModifiedBefore(Date modificationTime) throws FileStorageEx
     @Override
     public void processFileContent(FileDownloadProcessor fileDownloadProcessor) throws FileStorageException {
         FileEntry fileEntry = fileDownloadProcessor.getFileEntry();
-        InputStream fileContentStream = null;
-        try {
-            Blob blob = blobStore.getBlob(container, fileEntry.getId());
-            if (blob == null) {
-                throw new FileStorageException(
-                    MessageFormat.format(Messages.FILE_WITH_ID_AND_SPACE_DOES_NOT_EXIST, fileEntry.getId(), fileEntry.getSpace()));
-            }
-            Payload payload = blob.getPayload();
-            fileContentStream = payload.openStream();
+        Blob blob = blobStore.getBlob(container, fileEntry.getId());

```

Comment:
This here throws ContainerNotFoundException which is a runtime exception, previously was wrapped in FileStorageException (a checked exception). In our code, there are specific places where we catch and handle FileStorageException, rethrowing with a more meaningful message. With this change, the wrapping no longer occurs, hence the inability to enrich the message later.